docs/primer/whatdoesbodscover.rst

```diff
@@ -43,4 +43,4 @@ In order to enable worldwide use of the Standard, it does not directly define th
 
 .. highlights:: 
     
-    The Standard is well-defined yet flexible: as suited for use by national registers of beneficial ownership as by corporate researchers.
+    The Standard is well-defined yet flexible. National registers of beneficial ownership can use it; so can people researching or representing corporate structures.
```

Comment:
"representing corporate structures" could mean describing them or acting on behalf of them. I know what you mean, and I think the reader can work it out, but I think this might read better as "so can people researching corporate structures or those who need to describe them using BODS" or similar

Reply:
Have replaced 'representing' with 'describing'.